api/mfa_test.go

```diff
@@ -62,31 +88,31 @@ func (ts *MFATestSuite) TestEnrollFactor() {
 	}{
 		{
 			"TOTP: No issuer",
-			"john",
-			models.TOTP,
+			alternativeFriendlyName,
+			models.TOTP.String(),
 			"",
 			http.StatusOK,
 		},
 		{
 			"Invalid factor type",
-			"bob",
+			testFriendlyName,
 			"",
-			"john.com",
+			ts.TestDomain,
 			http.StatusUnprocessableEntity,
```

Comment:
will change to bad request

Reply:
For paper trail/context I initially set it as 422 since the [json is valid and it sounded like 422 might be more fitting](https://stackoverflow.com/questions/16133923/400-vs-422-response-to-post-of-data)

I think it's fine as 400 though, going to change